gluten-data/src/main/java/org/apache/gluten/memory/nmm/NativeMemoryManagers.java

```diff
@@ -48,6 +50,15 @@ public static NativeMemoryManager contextInstance(String name) {
         id, () -> createNativeMemoryManager(name, Collections.emptyList()));
   }
 
+  public static NativeMemoryPool arrowPool(String name) {
+    if (!TaskResources.inSparkTask()) {
+      throw new IllegalStateException("This method must be called in a Spark task.");
+    }
+    String id = "ArrowNativeMemoryPool:" + name;
+    return TaskResources.addResourceIfNotRegistered(id, () -> createArrowNativeMemoryPool(name))
+        .getArrowPool();
+  }
```

Comment:
Sorry, just missed this code.

Let's move the method (as well as `createArrowNativeMemoryPool`) to `ArrowNativeMemoryPool.scala`, thanks.